packages/backend/src/ee/services/ai/agents/tests/agent.integration.test.ts

```diff
@@ -175,10 +170,18 @@ describeOrSkip('agent integration tests', () => {
                 factualityEvaluation.answer === 'A' ||
                 factualityEvaluation.answer === 'B';
 
+            const toolsEvaluation = await llmAsJudgeForTools({
+                query: promptQueryText,
+                toolCalls,
+                expectedOutcome:
+                    'It should have used the findExplores tool to find the data models and then the table tool to get the data models',
```

Comment:
let's clarify "table tool"

Reply:
ooooh good catch, this is now deprecated since the agent has access to explores without the need of tool calls. I'll update